deploy/docker/Dockerfile

```diff
@@ -1,18 +1,6 @@
-FROM alpine:3.2
-MAINTAINER vishnuk@google.com
```

Comment:
The maintainer statement is deprecated https://github.com/docker/docker/pull/25466, also we just did remove all MAINTAINER statements from core k8s dockerfiles